.vscode/tasks.json

```diff
@@ -1,33 +1,13 @@
 {
 	"version": "2.0.0",
 	"tasks": [
-		{
-			"type": "func",
```

Comment:
I don't think we want to delete this one

Reply:
I tested without this and it works fine. Do we need a separate task for running the function?

Reply:
I think so for ingestion

Reply:
It looks like the task comes built-in with the VS Code extension

I have a different issue though, which is that the task isn't using the venv that poetry is installing dependencies to
![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/60179183/a18a1e9b-7ce5-42a5-a3bd-6a82aa123b82)

Reply:
Today I learnt - I've just got Functions attached and it even asked me what the Storage account was! Looks like they have improved the extension a lot